sensor/common/sensor/central_sender_impl.go

```diff
@@ -95,6 +99,12 @@ func (s *centralSenderImpl) send(stream central.SensorService_CommunicateClient,
 			log.Info("Context done")
 			s.stopper.Flow().StopWithError(stream.Context().Err())
 			return
+		case err := <-streamErrC:
+			if err != nil {
+				log.Infof("Error on sending to stream: %s", err)
```

Comment:
I guess that should not be `log.Info`. Did you mean `log.Errorf` or `log.Warnf`?

Reply:
Yeah... I've copy-pasted without paying to much attention...